### PR TITLE
Feature/cjc8/update lithooperators

### PR DESCRIPTION
**Pull request by cjc8**

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -4,13 +4,13 @@ platform :ios, '13.0'
...
   pod 'LUX', :path => '../'
   pod 'PlaygroundVCHelpers', git: 'https://github.com/ThryvInc/playground-vc-helpers'
-  pod 'FunNet/Combine', git: 'https://github.com/schrockblock/funnet'
-  pod 'FunNet/Multipart', git: 'https://github.com/schrockblock/funnet'
-  pod 'Slippers', git: 'https://github.com/ThryvInc/slippers'
-  pod 'LithoOperators', git: 'https://github.com/ThryvInc/LithoOperators'
-  pod 'FlexDataSource', git: 'https://github.com/ThryvInc/flex-data-source'
-  pod 'fuikit', git: 'https://github.com/ThryvInc/fuikit'
-  pod 'LithoUtils/Core', git: 'https://github.com/ThryvInc/litho-utils'
+  pod 'FunNet/Combine', git: 'https://github.com/schrockblock/funnet', branch: 'feature/cjc8/update-lithooperators'
```
Comment (cjc8) [inline Example/Podfile:7]: as with all of these, will remove branch req once everything is merged in

File: LUX/Classes/CollectionViews/LUXCollectionViewModel.swift [hunk 2]
```diff
@@ -122,8 +122,8 @@ public func pageableCollectionViewModel<T, U, C>(_ call: CombineNetCall,
         let dataPub = call.publisher.$data.eraseToAnyPublisher()
         let modelPub = unwrappedModelPublisher(from: dataPub, modelUnwrapper)
         let pageManager = LUXPageCallModelsManager(call, modelPub)
-        let modelToItem = configurer >||> LUXModelCollectionItem<U, C>.init
-        let modelsToItems = modelToItem >||> map
+        let modelToItem = configurer -*> LUXModelCollectionItem<U, C>.init
+        let modelsToItems = modelToItem -*> map
```
Comment (schrockblock) [inline LUX/Classes/CollectionViews/LUXCollectionViewModel.swift:126]: There's a newer version of map that takes a function from T to T and returns a `([T]) -> [T]`, we should use that here and get rid of the operator altogether (I know that wasn't the assignment per se, just, while we're here, ya know)

File: LUX/Classes/CollectionViews/LUXCollectionViewModel.swift [hunk 3]
```diff
@@ -122,8 +122,8 @@ public func pageableCollectionViewModel<T, U, C>(_ call: CombineNetCall,
         let dataPub = call.publisher.$data.eraseToAnyPublisher()
         let modelPub = unwrappedModelPublisher(from: dataPub, modelUnwrapper)
         let pageManager = LUXPageCallModelsManager(call, modelPub)
-        let modelToItem = configurer >||> LUXModelCollectionItem<U, C>.init
-        let modelsToItems = modelToItem >||> map
+        let modelToItem = configurer -*> LUXModelCollectionItem<U, C>.init
```
Comment (schrockblock) [inline LUX/Classes/CollectionViews/LUXCollectionViewModel.swift:125]: Could we use `modelItem` now?

**Review by schrockblock — APPROVED** [review]
This works, just some somewhat unrelated changes that may obviate some of the operators, which would be cool

**Pull request merged**